blog/20140130-mate-desktop-live-cd.de.md

```diff
@@ -26,27 +26,27 @@ Wenn Sie die zusätzlichen Bytes verschmerzen können, lassen Sie bitte den Clie
 nachdem der Download beendet ist, damit Sie es an andere zurück "seeden" können.
 *Für höchste Download-Geschwindigkeit wird ein "web-seed-fähiger" Client empfohlen.
 
-  * [matelivecd-14.069.1449.iso.torrent](http://repo.mate-desktop.org/livecd/matelivecd-14.069.1449.iso.torrent)
+  * [matelivecd-14.069.1449.iso.torrent](https://repo.mate-desktop.org/livecd/matelivecd-14.069.1449.iso.torrent)
```

Comment:
dead link

Reply:
I didn't found the live-cd or the torrent on server, so i will remove all links.